src/containers/Journal/index.js

```diff
@@ -1,8 +1,100 @@
-import React from 'react';
-import UnderConstruction from '../UnderConstruction';
+import React, { Component } from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import { Icon, Popconfirm } from 'antd';
 
-const Journal = props => {
-  return <UnderConstruction {...props} />;
+import data from './fakeData';
+import BackButton from '../../components/BackButton';
+import JournalComponent from '../../components/Journal';
+
+import './journal.css';
+
+class Journal extends Component {
+  state = {
+    journal: { ...data },
+    history: '',
+  };
+
+  handleConfirm = e => {
+    // it will delete the journal from firebase
+    const { history } = this.state;
+    history.push('/home');
+  };
+
+  handleGoBack = e => {
+    const { history } = this.state;
+    history.push('/home');
+  };
+
+  render() {
+    const { journal } = this.state;
+    return (
+      <div className="journal-page">
+        <div className="journal-page__header">
+          <BackButton handleBack={this.handleGoBack} />
+          <div>
+            <Popconfirm
+              title="Do you really want to delete this entry?"
+              onConfirm={this.handleConfirm}
+              okText="Yes"
+              cancelText="cancel"
+            >
+              <Icon type="delete" className="header__icon" />
+            </Popconfirm>
+          </div>
+        </div>
+        <div className="journal-card__top">
+          <div className="journal-card__date">
+            <Icon type="calendar" className="journal-card__icon" />
+            <span>
+              {
+                moment('2019-10-30T09:17:27.037Z')
```

Comment:
Is there a reason you want to default to this specific time? You might just want to get the current time like:
```suggestion
                moment()
```

Reply:
I'm supposed to use it in this way because I will receive it like this from firebase. But it was only for styling purposes  :+1: